### ISSUE TITLE
me want old ui

### ISSUE THREAD

**Issue by animzsans** [X8X-XZDX/VALVE-3RD-TURN#2]
me want old steam ui from old version of steam from around 2015.. will tickle ur feet for old steam ui :D

**Comment by X8X-XZDX** [comment]
Give me a magic switch that lets me set the UI version so I can have the god forsaken green steam UI from 2002.

I want the authentic CONNECTION SYNC for 3 hours too while TF2 has already loaded LOL

**Comment by animzsans** [comment]
> Give me a magic switch that lets me set the UI version so I can have the god forsaken green steam UI from 2002.
> 
> I want the authentic CONNECTION SYNC for 3 hours too while TF2 has already loaded LOL

yes also valve pls let me downgrade steam without you being a fat bitch and telling me to upgrade to modern steam slop
